php/src/Config.h

```diff
@@ -62,45 +29,31 @@ extern "C"
 // including php.h
 //
 #ifndef NDEBUG
-#define TMPDEBUG
+#   define TMPDEBUG
 #endif
 
 #include "php.h"
-
-#ifdef _WIN32
-#   pragma warning( default : 4018)
-#endif
-
 #include "php_ini.h"
 #include "ext/standard/info.h"
 #include "zend_interfaces.h"
 #include "zend_exceptions.h"
 
-#ifdef _WIN32
-}
-#endif
-
 //
 // Enable asserts if the extension is built with debug. It's fine to include several times
 // assert.h with a different NDEBUG setting.
 //
 #ifdef TMPDEBUG
-#undef TMPDEBUG
-#undef NDEBUG
-#include <assert.h>
+#   undef TMPDEBUG
+#   undef NDEBUG
+#   include <assert.h>
 #endif
 
 extern zend_module_entry ice_module_entry;
 #define phpext_ice_ptr &ice_module_entry
 
-#ifdef PHP_WIN32
-#define PHP_ICE_API __declspec(dllexport)
-#else
-#define PHP_ICE_API
-#endif
-
+// TODO what is this for?
```

Comment:
I think this is a leftover include from the php5 mapping that had some kind of php thread safe variant that I think was all removed in PHP 7